[2023-02-08 00:48:48,443] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: 09_Xcoms.task_3_pull scheduled__2023-01-29T00:00:00+00:00 [queued]>
[2023-02-08 00:48:48,457] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: 09_Xcoms.task_3_pull scheduled__2023-01-29T00:00:00+00:00 [queued]>
[2023-02-08 00:48:48,457] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-08 00:48:48,458] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-08 00:48:48,458] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-08 00:48:48,479] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): task_3_pull> on 2023-01-29 00:00:00+00:00
[2023-02-08 00:48:48,483] {standard_task_runner.py:52} INFO - Started process 53324 to run task
[2023-02-08 00:48:48,489] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', '09_Xcoms', 'task_3_pull', 'scheduled__2023-01-29T00:00:00+00:00', '--job-id', '1365', '--raw', '--subdir', 'DAGS_FOLDER/09_xcoms.py', '--cfg-path', '/tmp/tmpl2864ptd', '--error-file', '/tmp/tmp0cki_d4v']
[2023-02-08 00:48:48,491] {standard_task_runner.py:80} INFO - Job 1365: Subtask task_3_pull
[2023-02-08 00:48:48,582] {task_command.py:371} INFO - Running <TaskInstance: 09_Xcoms.task_3_pull scheduled__2023-01-29T00:00:00+00:00 [running]> on host pc.
[2023-02-08 00:48:48,680] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=09_Xcoms
AIRFLOW_CTX_TASK_ID=task_3_pull
AIRFLOW_CTX_EXECUTION_DATE=2023-01-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-29T00:00:00+00:00
[2023-02-08 00:48:48,689] {logging_mixin.py:115} INFO - 9
[2023-02-08 00:48:48,690] {python.py:173} INFO - Done. Returned value was: None
[2023-02-08 00:48:48,699] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=09_Xcoms, task_id=task_3_pull, execution_date=20230129T000000, start_date=20230208T054848, end_date=20230208T054848
[2023-02-08 00:48:48,740] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-08 00:48:48,765] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
